docs/guide/getting-started/basic-concepts.md

```diff
@@ -136,6 +136,8 @@ Sublime Text and plugins push debug information
 to the *console*.
 To open the console,
 press <kbd>Ctrl+\`</kbd>
+> **Tip:** The backtick key (<code>`</code>) is usually located to the left of the **1** key.  
+> On macOS, if <kbd>Ctrl</kbd>+<kbd>`</kbd> does not work due to keyboard layout, use **View → Show Console** from the main menu.
```

Comment:
Please see [CONTRIBUTING.md](https://github.com/sublimetext-io/docs.sublimetext.io/blob/master/CONTRIBUTING.md) on how we use custom blocks, shortcut keys and menu item captions.